bazel/repositories.bzl

```diff
@@ -527,3 +533,19 @@ def _com_github_google_jwt_verify():
         name = "jwt_verify_lib",
         actual = "@com_github_google_jwt_verify//:jwt_verify_lib",
     )
+
+def _apply_dep_blacklist(ctxt, recipes):
+    newlist = []
+    skip_list = []
+    if _is_linux_ppc(ctxt):
+        skip_list = PPC_SKIP_TARGETS
+    for t in recipes:
+        if t not in skip_list:
+            newlist.append(t)
+    return newlist
+
+def _is_linux_ppc(ctxt):
+    if ctxt.os.name != "linux":
+        return False
+    res = ctxt.execute(["uname", "-m"])
+    return "ppc" in res.stdout
```

Comment:
@dslomov is this the best practice we can use right now for achieving this? Any thoughts on how we can condition external dependencies in a cleaner way?